arch/arm/core/fault.c

```diff
@@ -463,7 +492,7 @@ static void _DebugMonitor(const NANO_ESF *esf)
  *
  * @return error code to identify the fatal error reason
  */
-static u32_t _HardFault(const NANO_ESF *esf)
+static u32_t _HardFault(NANO_ESF *esf)
```

Comment:
@andrewboie, @agross-linaro , this is a comment related to adding support for recoverable memory-protection faults for ARMv6-M/ARMv8-M Baseline:  is it _as easy_ as just calling the MemoryFaultIsRecoverable(.) function in line 501 (i.e. for ARMv6 only) and do similar things as @andrewboie does in Bus and MPU fault?

Reply:
We will also need to move the print-out of "*** HARD FAULT ***" below the call of _MemoryFaultIsRecoverable, to avoid any fault dumping in this case...

Reply:
I think at this point I've let go of my desire to not have any printouts for recoverable faults, it just makes the code excessively complicated.